src/Core/Configuration/ConnectedModeFeaturesConfiguration.cs

```diff
@@ -30,15 +30,22 @@ namespace SonarLint.VisualStudio.Core.Configuration
     public interface IConnectedModeFeaturesConfiguration
     {
         /// <summary>
-        /// Indicates whether Local Hotspot Analysis is supported in the current Connected Mode state  
+        /// Indicates whether Local Hotspot Analysis is supported in the current Connected Mode state
         /// </summary>
-        /// <returns>True if connected to SCloud or SQube 9.7 and above, False otherwise</returns>
```

Comment:
why did you remove this?

Reply:
I didn't mean to I think instead of copy paste I made cut paste by mistake.